qutebrowser/browser/network/proxy.py

```diff
@@ -61,9 +61,10 @@ def queryProxy(self, query):
         """
         proxy = config.val.content.proxy
         if proxy is configtypes.SYSTEM_PROXY:
-            # On Linux, use "export http_proxy=socks5://host:port" to manually
-            # set system proxy.
-            # ref. http://doc.qt.io/qt-5/qnetworkproxyfactory.html#systemProxyForQuery
+            # On Linux, use "export http_proxy=socks5://host:port"
```

Comment:
I think this might have been a mistake change, was this intentional?

Reply:
This can be dropped, it has been merged elsewhere IIRC.

Reply:
I fixed the line being too long, and you're reverting it here. You should drop it :wink:

Reply:
What's the best way to drop it?

Reply:
Since the change is only present in one commit (def2920), you can either:

1. `git revert def2920`, which will create a new commit reverting the change.
2. Use an interactive rebase to remove the above commit, which is a bit harder to do.
3. Create a new branch, cherry pick 132095c onto it (since it is the important commit). With this option, you'll have to create a new PR for the new branch.

If you highlight me on irc, I can walk you through one of these if you'd like as well!